source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,401 @@
+* 40 Book_Segments and/or Reference_Items, 10 each from 4 different Books and/or Reference_Works with Access_Type Controlled.
+* 40 Book_Segments and/or Reference_Items, 10 each from 4 different Books and/or Reference_Works with Access_Type Open.
+* 20 Book_Segments and/or Reference_Items, 10 each from 2 different Books and/or Reference_Works with Access_Type Free_To_Read.
 
-This must result in:
+This MUST result in 40 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 4 each of Unique_Title_Investigations and Unique_Title_Requests with Access_Type Controlled; the same again for Access_Type Open; and 20 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 2 each of Unique_Title_Investigations and Unique_Title_Requests with Access_Type Free_To_Read.
 
-* 100 Total_Item_Requests being reported in the TR_B1 Standard View.
-* 20 Unique_Title_Requests being reported in the TR_B1 Standard View.
+**Option 3**: Report provider offers Book_Segments and/or Reference_Items under only one Access_Type.
 
+The auditor MUST request
 
-Audit Test B1-2: Total_Item_Requests and Unique_Title_Requests - 30-second filters
-..................................................................................
+70 Book_Segments and/or Reference_Items, 10 each from 7 different Books and/or Reference_Works.
 
-The audit test consists of clicking links to an item within a book twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Requests must be recorded. If the two clicks occur with more than 30 seconds between, then 2 Total_Item_Requests must be counted. In both cases only 1 Unique_Title_Requests will be reported.
+This MUST result in 70 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests, and 7 each of Unique_Title_Investigations and Unique_Title_Requests with the appropriate Access_Type.
 
-The auditor must carry out a total of 32 tests, and each test will consist of 2 requests. There are 2 types of tests that must be carried out:
 
-* “Inside” tests (Two requests are made to the same Item, and the second request is made within 30 seconds of the first).
-* “Outside” tests (Two requests are made to the same item and the second request is made more than 30 seconds after the first).
+E.2.4.3 Book Access Types: Whole Books and/or Reference_Works
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-The auditor must carry out 16 **inside** tests.
+Within the Title Report, breakdowns by Access_Type are essential. There are therefore a series of audit tests designed to determine report providers’ compliance with requirements for reporting Access_Type. These tests only apply where Books with more than one Access_Type are available on a platform, and they are only available without Book_Segments.
```

Comment:
The "only apply where Books with more than one Access_Type are available" no longer is correct.